yoti/plugins/yotiprofile/language/en-GB/en-GB.plg_user_yotiprofile.ini

```diff
@@ -17,6 +17,9 @@ PLG_USER_YOTIPROFILE_FIELD_SELFIE_FILENAME_LABEL="Selfie"
 PLG_USER_YOTIPROFILE_FIELD_GIVEN_NAMES_LABEL="Given Names"
 PLG_USER_YOTIPROFILE_FIELD_GIVEN_NAMES_DESC="User Given names"
 PLG_USER_YOTIPROFILE_FIELD_GIVEN_NAMES_MESSAGE="User Given names"
+PLG_USER_YOTIPROFILE_FIELD_FULL_NAME_LABEL="Full Name"
+PLG_USER_YOTIPROFILE_FIELD_FULL_NAME_DESC="User Full Name"
+PLG_USER_YOTIPROFILE_FIELD_FULL_NAME_MESSAGE="User FULL Name"
```

Comment:
Is there a reason this is in caps? Doesn't look like any of the other strings are in caps

Reply:
No reason, i will change that.